sync_labels.py

```diff
@@ -5,6 +5,7 @@
 import argparse
 from github import (
     Github,
+    GithubException,
```

Comment:
Should this be part of the first commit commit that reduces redundancy?

Reply:
Nice catch. Actually, it should be totally removed. Catching just the `GithubException` was too limiting. Other exceptions were being raised that halted the whole process when it should have just carried on.